src/scripts/main.js

```diff
@@ -1,3 +1,36 @@
 'use strict';
 
-// write your code here
+let elements = [...document.querySelectorAll('.population')];
+
+elements = elements.map((e) => {
+  return +e.textContent.split(',').join('');
+});
+
+let sum = elements.reduce((totall, element) => element + totall, 0);
```

Comment:
There is a typo in the variable name 'totall'. It should be 'total'. This could lead to confusion and should be corrected.